docs/source/profiler.rst

```diff
@@ -281,6 +281,18 @@ Saving and Loading a Profile
 
 The profiles can easily be saved and loaded as shown below:
 
+There are two save/load methods:
+
+* **pickle save/load**
+    * save a profile as a pkl file.
+    * load a pkl file as a profile object.
+
+* **json save/load**
+    * save a profile as a human-readable json file.
+    * load a json file as a profile object.
+
+**NOTE: json saving and loading isn't supported for unstructured profiles.**
+
 .. code-block:: python
```

Comment:
I'd recommend another code-block too for JSON save / load example too ... could just write up an example since we know top-level API

Reply:
done